### PR TITLE
Improve documentation on Elasticsearch index config

### PR DESCRIPTION
**Pull request by simonswine**
This will raise the number of fields allowed in JSON document.

Signed-off-by: Christian Simon <simon@swine.de>

```release-note
NONE
```
/assign @JoshVanL

### CODE REVIEW

File: docs/user-guide.rst [hunk 1]
```diff
@@ -425,6 +425,22 @@ Both of those outputs can then be used in the tarmak configuration:
...
 
+Configuring Index Templates
++++++++++++++++++++++++++++
+
+Fluentbit will publish into a new index everyday.  To optimise all of those
+indices for our logging purpose, it is beneficial to adapt some settings
+through index templates. We suggest at least considering raising the field
+limit from the default of 1000.  Also depending on the size of the
+Elasticsearch installation the number of shards and replicas should be adapted.
+This example is suggested for a here for a single node setup:
```
Comment (JoshVanL) [inline docs/user-guide.rst]: "This example is suggested here for a single node setup:" ?

Comment (simonswine) [reply docs/user-guide.rst]: Good catch

**Pull request merged**